language-plutus-core/language-plutus-core.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version: 1.18
+cabal-version: 2.0
```

Comment:
TBH, I am not sure whether that is a wise move without coordinating with the Cardano SL devs. At the end of the day, our code has to integrate with theirs (and they have the much bigger code base). So, please run this by, say, @erikd to make sure we won't get into issue when integrating.